client/src/pages/SubmissionDraft.vue

```diff
@@ -79,8 +79,9 @@
           <section class="q-mt-lg">
             <p>{{ $t(`submissions.create.submit.description`) }}</p>
             <q-btn
+              :disabled="draft.content.required.$invalid"
```

Comment:
![image](https://github.com/MESH-Research/Pilcrow/assets/463159/41c8df6e-030c-4c83-841a-b520d368f359)
Disabling the button appears to leave it in a undesirable state (see screenshot). Browser: Chrome

Reply:
Good catch. I've been testing in dark mode this whole time and never noticed this.

Reply:
This will now appear as expected. I also made the "Go" button the primary color to be consistent with the active call-to-action visual pattern throughout the application.